cpp/benchmarks/CMakeLists.txt

```diff
@@ -20,8 +20,8 @@ target_compile_options(
                         "$<$<COMPILE_LANGUAGE:CUDA>:${RAPIDSMPF_CUDA_FLAGS}>"
 )
 target_link_libraries(
-  bench_shuffle PRIVATE rapidsmpf::rapidsmpf MPI::MPI_C ucxx::ucxx ucx::ucp
-                        $<TARGET_NAME_IF_EXISTS:conda_env>
+  bench_shuffle PRIVATE rapidsmpf::rapidsmpf ucxx::ucxx ucx::ucp
+                        $<TARGET_NAME_IF_EXISTS:MPI::MPI_C> $<TARGET_NAME_IF_EXISTS:conda_env>
```

Comment:
Not a big deal, but do we even need to check whether MPI exists here, as well as in examples/tests? Currently benchmarks are always disabled if MPI is also disabled, making this check redundant. However, I can see the appeal of leaving this for future use when we have a standalone launcher for UCXX, for example.

Reply:
> I can see the appeal of leaving this for future use when we have a standalone launcher for UCXX

Yeah, that was my thinking